### DISCUSSION TITLE
Control Innerblocks in 3rd party components like Slick Slider

### DISCUSSION THREAD

**Discussion by iamsayan** [WordPress/gutenberg#44505] [Developer Experience]
![image](https://user-images.githubusercontent.com/25281493/192540782-965cc399-f6a2-4a84-b2b8-3713684cbdd1.png)

I am using the Slick Slider: https://react-slick.neostack.com/

here { children } is returning object, but I need to initialize the slider after the html output from { children } is already being loaded. and on every update on innerBlocks, we need to re-render the Slider Component.

Currently, it is outputting this type of structure:

![image](https://user-images.githubusercontent.com/25281493/192542007-8f7fe056-c55f-4fb7-9549-6b9ec149f7e8.png)

But I need to get this structure: 

![image](https://user-images.githubusercontent.com/25281493/192543364-e3ee481b-5933-4c87-b62c-51c1604676cf.png)

and also, re-render this if there is any inner block addition/deletion happens.

So, is there any way to get this? any hooks? or anything else?

Thank you.

**Comment by t-hamano** [comment]
This is not an answer about Gutenberg, but the issue submitted to that library may be helpful.

[Not working with custom component children.](https://github.com/akiran/react-slick/issues/591)
